<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RemoteLinux::Internal::RemoteLinuxDeployConfigurationWidget</class>
 <widget class="QWidget" name="RemoteLinux::Internal::RemoteLinuxDeployConfigurationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>617</width>
    <height>361</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Files to deploy:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTreeView" name="deploymentDataView"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
